emoji-ui/src/components/emojiItem.css

```diff
@@ -0,0 +1,18 @@
+.Emoji-container {
+  display: inline-block;
+  border-radius: 10px;
+  padding: 15px;
+}
+
+.Emoji-container:hover, .Emoji-container-selected {
+  background-color: #d4e8f9;
+}
+
+.Emoji-item {
+  width: 45px;
+  height: 45px;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center center;
```

Comment:
you can just write `center` once -- the other will default to center

Reply:
lol! where did that second `center` comes from! will remove 👍

Reply:
the second center is perfectly valid (for vertical vs horizontal positioning) -- it's just not needed so people usually drop it.